src/spec/expectations.cr

```diff
@@ -185,68 +185,90 @@ module Spec
     end
   end
 
+  # This module defines a number of methods to create expectations, which are
+  # automatically included into the top level namespace.
   module Expectations
+    # Creates an `Expectation` that passes if actual equals *value* (`==`).
     def eq(value)
       Spec::EqualExpectation.new value
     end
 
+    # Creates an `Expectation` that passes if actual and *value* are identical (`.same?`).
     def be(value)
       Spec::BeExpectation.new value
     end
 
+    # Creates an `Expectation` that passes if actual is true (`== true`).
     def be_true
       eq true
     end
 
+    # Creates an `Expectation` that passes if actual is false (`== false`).
     def be_false
       eq false
     end
 
+    # Creates an `Expectation` that passes if actual is truthy (neither `nil` nor `false`).
     def be_truthy
       Spec::BeTruthyExpectation.new
     end
 
+    # Creates an `Expectation` that passes if actual is falsy (`nil` or `false`).
     def be_falsey
       Spec::BeFalseyExpectation.new
     end
 
+    # Creates an `Expectation` that passes if actual is nil (`== nil`).
     def be_nil
       eq nil
     end
 
+    # Creates an `Expectation` that passes if actual is within *delta* of *expected*.
     def be_close(expected, delta)
       Spec::CloseExpectation.new(expected, delta)
     end
 
+    # Returns a factory to create a comparison `Expectation` that:
+    #
+    # * passes if actual is lesser than *value*: `be < value`
+    # * passes if actual is lesser than or equal *value*: `be <= value`
+    # * passes if actual is greater than *value*: `be > value`
+    # * passes if actual is greater than or equal *value*: `be >= value`
     def be
       Spec::Be
     end
 
+    # Creates an `Expectation` that passes if actual matches *value* (`=~`).
     def match(value)
       Spec::MatchExpectation.new(value)
     end
 
-    # Passes if actual includes *expected*. Works on collections and `String`.
+    # Creates an `Expectation` that  passes if actual includes *expected* (`.includes?`).
+    # Works on collections and `String`.
     def contain(expected)
       Spec::ContainExpectation.new(expected)
     end
 
+    # Creates an `Expectation` that passes if actual is of type *type* (`is_a?`).
     macro be_a(type)
       Spec::BeAExpectation({{type}}).new
     end
 
+    # Runs the block and passes if it raises an exception.
```

Comment:
Worth noting about this method is that this variant that doesn't verify it against an actual exception type has been deprecated in rspec. From the places where I fixed the deprecation errors for that, I can say that the reason for the deprecation was very motivated, as there was lots of hidden errors and unexpected exceptions being caught.

Reply:
Good point! I'll add a note about this and suggest removing the matcher in a separate issue.